src/main/java/com/lambdaschool/foundation/controllers/CityController.java

```diff
@@ -33,13 +33,13 @@
      *  resources are available)
      * @return list of all cities
      */
-//    @GetMapping(value = "/all",
-//       produces = "application/json")
-//    public ResponseEntity<?> listAllCities()
-//    {
-//        List<City> cities = cityService.findAll();
-//        return  new ResponseEntity<>(cities, HttpStatus.OK);
-//    }
+    @GetMapping(value = "/all",
```

Comment:
Thanks for commenting this function back in. It will be useful on the front end.